spec/rspec/core/configuration_spec.rb

```diff
@@ -1229,7 +1229,171 @@ def metadata_hash(*args)
       end
     end
 
+    describe "color options" do
+      before { config.no_color = false }
+
+      context "#force_color=false, #no_color=false" do
```

Comment:
This assumes that these options default to false.  I think it'd be better to express that assumption with some expectations in a `before` hook (or set them explicitly and do not assume it at all).